aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1291,6 +1293,7 @@ def __init__(
         self._strip_prefixes = strip_prefixes
         self.context: Dict = {}  # early init as customers might add context before event resolution
         self.processed_stack_frames = []
+        self.response_builder_class = ResponseBuilder
```

Comment:
any particular reason to have our ResponseBuilder as a public attribute?

Reply:
Moved it to a private attribute `self._response_builder_class`